xml/Microsoft.Extensions.Configuration/CommandLineConfigurationExtensions.xml

```diff
@@ -94,7 +94,7 @@
         <param name="configurationBuilder">The <see cref="T:Microsoft.Extensions.Configuration.IConfigurationBuilder" /> to add to.</param>
         <param name="args">The command line args.</param>
         <summary>
-              Adds a <see cref="T:Microsoft.Extensions.Configuration.CommandLine.CommandLineConfigurationProvider" /> <see cref="T:Microsoft.Extensions.Configuration.IConfigurationProvider" /> 
+              Adds a <see cref="T:Microsoft.Extensions.Configuration.CommandLine.CommandLineConfigurationProvider" /><see cref="T:Microsoft.Extensions.Configuration.IConfigurationProvider" /> 
```

Comment:
@huangmin-ms This seems like a bug in mdoc that it removed this space? (Same for the next 5 changes.)

Reply:
@gewarren  This is because in mdoc we use [XmlDocument.Load](https://learn.microsoft.com/en-us/dotnet/api/system.xml.xmldocument.load?view=net-6.0#system-xml-xmldocument-load(system-string)) without setting [PreserveWhitespace](https://learn.microsoft.com/en-us/dotnet/api/system.xml.xmldocument.preservewhitespace?view=net-6.0) as `true`.  I tried to set this flag but the output xml was impacted. please open a bug for us to do further investigation.

Reply:
@gewarren  this is because the default value of  [PreserveWhitespace](https://learn.microsoft.com/en-us/dotnet/api/system.xml.xmldocument.preservewhitespace?view=net-6.0) is `false` when loading XML Document.
Please help to open a bug for us to track this issue.

Reply:
I logged https://dev.azure.com/ceapex/Engineering/_workitems/edit/738082.